civismlext/test/test_preprocessing.py

```diff
@@ -677,6 +635,22 @@ def test_pickle(data_raw, dataframe_expected):
     expander = pickle.load(buff)
 
     arr = expander.transform(data_raw)
-    expected_array = np.asarray(dataframe_expected)
     assert arr.shape == expected_array.shape
     assert_almost_equal(arr, expected_array)
+
+
+def test_categorical_looks_like_int():
```

Comment:
This test fails before the changes in this PR.

Reply:
Should we include a mixed-type categorical here as well?

Reply:
yes!

Reply:
You're right, we should. Added!